docs/restore-individual-tables.md

```diff
@@ -1,41 +1,50 @@
 # Restore individual tables
 
-Percona XtraBackup can export individual tables from any InnoDB database, if they are individual `.ibd` files, and import them into Percona Server for MySQL with XtraDB or MySQL 8.0. The source doesn’t have to be XtraDB or MySQL 8.0, but the destination must be.
+You can use Percona XtraBackup to move specific tables from one database to another. This works for InnoDB tables that are stored as separate `.ibd` files. You can move these tables into either Percona Server for MySQL with XtraDB or MySQL 8.0. Your source database can be any type, but your destination must be XtraDB or MySQL 8.0.
 
-The example exports and imports the following table:
+When you do this, you'll work with several types of files:
 
-```text
-CREATE TABLE export_test (
+| File                             | Description |
+|----------------------------------|-------------|
+| ibdata1                          | This is the system tablespace file in InnoDB. It contains critical metadata and potentially data for various tables. It's often required for any InnoDB restore, including single table restores. |
+| undo_001 and undo_002            | These are the undo log files. They are necessary for transaction rollback and crash recovery during the prepare phase. They are required for a consistent backup. |
+| mysql.ibd                        | This file contains the InnoDB data dictionary. It's crucial for understanding the structure of your tables and is essential for restoring individual tables. |
+| xtrabackup_info                  | This file contains metadata about the backup itself, such as the server version and backup time. |
+| xtrabackup_checkpoints           | This file records checkpoint information, which is used for crash recovery and redo log application. |
+| xtrabackup_logfile               | This is the redo log file, which records all changes made to the database. It's essential for bringing the backup to a consistent state. |
+| xtrabackup_tablespaces           | This file lists all tablespaces that were backed up. |
```

Comment:
The list of tablspaces that are stored external to the datadir. On restore, xtrabacup will try to restore the files to same locations as recorded in the file.